index.html

```diff
@@ -87,6 +91,8 @@
             <br>
         </div>
         <br>
+        <div class="plyr"><audio></audio></div>
```

Comment:
> Don't worry about it right now :smile:  

So I forgot about it. And the same bug I had before happened again. I spent about an hour trying to figure it out before I remembered about this. More of a rant than something we should discuss. :stuck_out_tongue_winking_eye: